venvUtils/ensureVenv.py

```diff
@@ -88,13 +88,13 @@ def populate():
 			"--upgrade",
 			"pip",
 			"&&",
-			# py2exe is not compatible with setuptools 70+
-			# wheel must be manually installed when creating an non-isolated build with a custom setuptools version.
+			# py2exe is now compatible with setuptools 72.0
+			# There is now no need to manually install the wheel when creating non-isolated builds using a customized version of setuptools.
```

Comment:
`wheel` is a package installed separately to setuptools. This comment should not be changed. Please listen to suggestions. The previous suggestion just asked you to remove the line for setuptools

```suggestion
			# wheel must be manually installed when creating an non-isolated build with a custom setuptools version.
```